src/firewall/core/rich.py

```diff
@@ -434,11 +434,13 @@ def check(self, family=None):
 
 
 class Rich_Limit(_Rich_Entry):
-    def __init__(self, value):
+    def __init__(self, value, burst=None):
         self.value = value
+        self.burst = burst
```

Comment:
`self._burst = burst`

Reply:
no. This is intentional. so that the constructor does the normalization. As requested.